magenta/interfaces/midi/README.md

```diff
@@ -147,61 +151,22 @@ using VPMK and FluidSynth, your command would look like this:
 $ bazel-bin/magenta/interfaces/midi/magenta_midi \
   --input_port="VMPK Output" \
   --output_port="FluidSynth virtual port" \
-  --bundle_file=/tmp/attention_rnn.mag
+  --bundle_file=/tmp/attention_rnn.mag \
+  --phase_bars=4
```

Comment:
Nit: Do you mean "phrase" bars? Like how long the phrase will be? Phase usually denotes what part of a phrase you would be in, not the length of the phrase.